app/jobs/scholarsphere_pdf_file_version_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class ScholarspherePdfFileVersionJob < ApplicationJob
+  queue_as 'scholarsphere-pdf-file-version'
+
+  def perform(file_meta:, publication_meta:, exif_file_version:)
+    pdf_file_version = ScholarspherePdfFileVersion.new(file_meta: file_meta, publication_meta: publication_meta).version
+
+    Rails.cache.write("file_version_job_#{job_id}", { pdf_file_version: pdf_file_version, file_meta: file_meta, exif_file_version: exif_file_version })
```

Comment:
Admittedly, I don't have my head completely wrapped around everything that's going on in this PR, so maybe the answer is actually obvious, but I'm curious about what led to the decision to use Rails caching to store this information vs. using the database.

Also, what cache store are we planning to use in production?

Reply:
Currently, we are planning to use file caching in production for this work. However, since this is still a work in progress and is being tested on a preview branch, it will not be deployed just yet. We plan to revisit this in the next cycle.

Reply:
The more I think about this, the more I think my ignorance of rmd's data model may be getting in the way 

Perhaps we could add a `version` column to the work deposit, and use that as an indicator? 

I think that the reason we don't do this is because a work can have many files, and we need to make sure we've processed all the files before determining the final version. 

I'd be open to discuss this sometime soon, because I do think there's room for improvement perhaps.

Reply:
There's a couple reasons we don't use a version column on the work deposit:

- The version is ultimately determined by the user.  The automated version checking just prefills the form on the next page, so the user can confirm.
- The work deposit isn't actually created until the end of the workflow.  This could be changed, but then we'd have to come up with a mechanism for removing or restarting the created work deposit if a user leaves the workflow without finishing.

Reply:
Not saying we couldn't do it this way, but it might take a little (or a lot) of extra work.